日次CIエラーレート上昇: 水平スケール

直近2週間の水平スケールのE2Eテストでエラーレートが30%を超えている。(14回中5回同様のエラーが発生)

エラー内容:

```console
2022/04/28 04:19:10 step1-2: check results
2022/04/28 04:19:11 [ERROR] got unexpected status code: 503
    e2e_test.go:303: got unexpected status code: 503
```

--- a/e2e/horizontal_scaling/autoscaler.yaml
+++ b/e2e/horizontal_scaling/autoscaler.yaml
@@ -5,6 +5,8 @@ resources:
     min_size: 0
     max_size: 5
 
+    setup_grace_period: 30
+
     plans:
       - {name: "smallest", size: 0}
       - {name: "medium", size: 3}

